front/src/applications/operationalStudies/components/Study/AddOrEditStudyModal.tsx

```diff
@@ -79,58 +95,86 @@ export default function AddOrEditStudyModal({ editionMode, study, getStudy }: Pr
     setCurrentStudy({ ...currentStudy, tags: newTags });
   };
 
-  const createStudy = async () => {
-    if (!currentStudy?.name) {
+  const createStudy = () => {
+    if (!currentStudy) {
       setDisplayErrors(true);
     } else {
-      try {
-        const result = await post(`${rootURI}`, currentStudy);
-        dispatch(updateStudyID(result.id));
+      createStudies({
+        projectId: projectID as number,
+        studyCreateRequest: currentStudy as StudyCreateRequest,
+      }).then(() => {
```

Comment:
I think it's clearer when we use directly the result of the request like this (you can have a look at AddOrEditProjectModal.tsx)
```suggestion
      })
        .unwrap()
        .then((createdStudy) => {
```